UIKit/_UICache.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/NSObject.h>
+
+API_AVAILABLE(ios(7.0))
```

Comment:
I think you removed the class declaration here

Reply:
🤦🏻‍♂️